backend/src/ee/services/audit-log/audit-log-types.ts

```diff
@@ -2512,89 +2521,154 @@ interface LoadProjectKmsBackupEvent {
   metadata: Record<string, string>; // no metadata yet
 }
 
-interface CreateCertificateTemplate {
-  type: EventType.CREATE_CERTIFICATE_TEMPLATE;
+interface OrgAdminAccessProjectEvent {
+  type: EventType.ORG_ADMIN_ACCESS_PROJECT;
+  metadata: {
+    userId: string;
+    username: string;
+    email: string;
+    projectId: string;
+  }; // no metadata yet
+}
+
+interface OrgAdminBypassSSOEvent {
+  type: EventType.ORG_ADMIN_BYPASS_SSO;
+  metadata: Record<string, string>; // no metadata yet
+}
+
+interface CreateCertificateTemplateEstConfig {
+  type: EventType.CREATE_CERTIFICATE_TEMPLATE_EST_CONFIG;
   metadata: {
     certificateTemplateId: string;
+    isEnabled: boolean;
+  };
+}
+
+interface GetAzureAdCsTemplatesEvent {
+  type: EventType.GET_AZURE_AD_TEMPLATES;
+  metadata: {
     caId: string;
-    pkiCollectionId?: string;
-    name: string;
-    commonName: string;
-    subjectAlternativeName: string;
-    ttl: string;
+    amount: number;
   };
 }
 
-interface GetCertificateTemplate {
-  type: EventType.GET_CERTIFICATE_TEMPLATE;
+interface UpdateCertificateTemplateEstConfig {
+  type: EventType.UPDATE_CERTIFICATE_TEMPLATE_EST_CONFIG;
   metadata: {
     certificateTemplateId: string;
+    isEnabled: boolean;
+  };
+}
+
+interface GetCertificateTemplateEstConfig {
+  type: EventType.GET_CERTIFICATE_TEMPLATE_EST_CONFIG;
+  metadata: {
+    certificateTemplateId: string;
+  };
+}
+
+interface CreateCertificateTemplate {
+  type: EventType.CREATE_CERTIFICATE_TEMPLATE;
+  metadata: {
+    certificateTemplateId: string;
+    name: string;
+    projectId: string;
   };
 }
 
 interface UpdateCertificateTemplate {
   type: EventType.UPDATE_CERTIFICATE_TEMPLATE;
   metadata: {
     certificateTemplateId: string;
-    caId: string;
-    pkiCollectionId?: string;
     name: string;
-    commonName: string;
-    subjectAlternativeName: string;
-    ttl: string;
   };
 }
 
 interface DeleteCertificateTemplate {
   type: EventType.DELETE_CERTIFICATE_TEMPLATE;
   metadata: {
     certificateTemplateId: string;
+    name: string;
   };
 }
 
-interface OrgAdminAccessProjectEvent {
-  type: EventType.ORG_ADMIN_ACCESS_PROJECT;
+interface GetCertificateTemplate {
+  type: EventType.GET_CERTIFICATE_TEMPLATE;
+  metadata: {
+    certificateTemplateId: string;
+    name: string;
+  };
+}
+
+interface ListCertificateTemplates {
+  type: EventType.LIST_CERTIFICATE_TEMPLATES;
   metadata: {
-    userId: string;
-    username: string;
-    email: string;
     projectId: string;
-  }; // no metadata yet
+  };
 }
 
-interface OrgAdminBypassSSOEvent {
-  type: EventType.ORG_ADMIN_BYPASS_SSO;
-  metadata: Record<string, string>; // no metadata yet
+interface CreateCertificateProfile {
+  type: EventType.CREATE_CERTIFICATE_PROFILE;
```

Comment:
IGNORE PLACEMENT OF COMMENT

From a UX perspective, I think I should be able to issue certificates from here?
<img width="1511" height="855" alt="image" src="https://github.com/user-attachments/assets/3158d334-f56d-4c01-a88c-7f70c342803d" />